app/Repositories/Cts/ValidationPositionRepository.php

```diff
@@ -0,0 +1,40 @@
+<?php
+
+namespace App\Repositories\Cts;
+
+use App\Models\Cts\Member;
+use App\Models\Cts\ValidationPosition;
+use Carbon\Carbon;
+use Illuminate\Support\Collection;
+
+class ValidationPositionRepository
+{
+    public function findByPositionId(int $id)
+    {
+        return ValidationPosition::find($id)->first();
+    }
+
+    public function findByPosition(string $position)
```

Comment:
If you use findOrFail here, you can try/catch the ModelNotFound exception instead of checking that the return value is empty.

Reply:
e5d4add